docs/csharp/csharp-6.md

```diff
@@ -1,17 +1,570 @@
 ---
 title: What's New in C# 6 | C# Guide
-description: What's New in C# 6    
+description: Learn the new features in C# Version 6    
 keywords: .NET, .NET Core
-author:  tdykstra
+author:  BillWagner
 manager: wpickett
-ms.date: 06/20/2016
+ms.date: 09/22/2016
 ms.topic: article
-ms.prod: .net-core
-ms.technology: .net-core-technologies
-ms.devlang: dotnet
+ms.prod: visual-studio-dev-14
+ms.technology: devlang-csharp
+ms.devlang: csharp
 ms.assetid: 4d879f69-f889-4d3f-a781-75194e143400
 ---
 
-# What's New in C# 6    
-    
-For information about new features in C# 6, we suggest you head over to [the Roslyn repository in GitHub](https://github.com/dotnet/roslyn/wiki/New-Language-Features-in-C%23-6).
+# What's New in C# 6
+
+The 6.0 release of C# contained many features that improves
+productivity for developers. Features in this release include:
+
+* [Readonly Auto properties](#readonly-auto-properties)
+* [Auto Property Initializers](#auto-property-initializers)
+* [Expression Bodied function members](#expression-bodied-function-members)
+* [using static](#using-static)
+* [Null - conditional operators](#null-conditional-operators)
+* [String Interpolation](#string-interpolation)
+* [Exception filters](#exception-filters)
+* [nameof Expressions](#nameof-expressions)
+* [await in catch and finally blocks](#await-in-catch-and-finally-blocks)
+* [index initializers](#index-initializers)
+* [Extension methods for collection initializers](#extension-add-methods-in-collection-initializers)
+* [Improved overload resolution](#improved-overload-resolution)
+
+The overall effect of these features is that you write more concise code
+that is also more readable. The syntax contains less ceremony for many
+common practices. It's easier to see the design intent with less
+ceremony. Learn these features well, and you'll be more productive,
+write more readable code, and concentrate more on your core features
+than on the constructs of the language.
+
+The remainder of this topic provides details on each these features.
+
+## Auto Property enhancements 
+
+The syntax for auto-properties made it very easy to create properties
+that had simple get and set accessors:
+
+[!code-csharp[ClassicAutoProperty](../../samples/snippets/csharp/new-in-6/oldcode.cs#ClassicAutoProperty)]
+
+However, this simple syntax limited the kinds of designs you could support using
+auto properties. C# 6 improves the auto properties capabilities so that you can use
+them in more scenarios. You'll fall back on the more verbose syntax of declaring
```

Comment:
This last sentence reads a bit backward to me. Perhaps write it as 
"You won't need to fall back on the more verbose syntax of declaring and manipulating the backing field by hand so often."
Noticed I shortened a bit as well.